app/models/course/assessment/submission.rb

```diff
@@ -128,6 +128,10 @@ class Course::Assessment::Submission < ApplicationRecord
   #   Returns submissions which have been submitted (which may or may not be graded).
   scope :confirmed, -> { where(workflow_state: [:submitted, :graded, :published]) }
 
+  # @!method self.published
+  #   Returns submissions which have been published.
+  scope :published, -> { where(workflow_state: :published) }
```

Comment:
Can you check if workflow adds this scope already? I thought it does.

Reply:
No it doesn't.

`Course::Assessment::Submission.published` doesn't work on the current master branch (without this scope).

Reply:
The scope defined by workflow is `with_published_state`

Reply:
oops. for future reference

https://github.com/geekq/workflow#scopes